### PR TITLE
added OS versions

### PR DESCRIPTION
**Pull request by asenkowski**
## **Please check that the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CI passes

### CODE REVIEW

**Comment by cpheps** [comment]
@asenkowski Going to submit a separate PR to fix the linter issue

File: README.md [hunk 1]
```diff
@@ -41,6 +41,28 @@ Utilize Plugins to get up and running quickly. Here's a quick list of Stanza's m
...
+- 2012 r2
+- 2016
+- 2019
+- 2022
+
+### Linux
+- Ubuntu 16.04, 18.04, 20.04
+- Debian 9, 10, 11
+- Alma, Rocky 8
+- Red Hat 7, 8
```
Comment (jsirianni) [inline README.md:57]: I think we can call out centos 7 and centos 8 stream

**Review by jsirianni — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (asenkowski) [reply README.md:57]: added those in

**Comment by cpheps** [comment]
@asenkowski Should be good if you rebase or merge in main

**Review by jsirianni — APPROVED** [review]

**Review by cpheps — APPROVED** [review]

**Pull request merged**